build.gradle

```diff
@@ -32,6 +32,7 @@ buildscript {
             'strikt':'0.33.0',
```

Comment:
I don't think I see the coroutines dependency here, where is that version being set?

Reply:
The coroutine dependency is coming from compose itself that exposes it. I think it uses the version that works with the version of Kotlin that compose is built against.